Kopernicus/Kopernicus/Configuration/Parser/BuiltinTypeParsers.cs

```diff
@@ -142,7 +142,7 @@ public void SetFromString(string s)
 
                 if (s.StartsWith("BUILTIN/"))
                 {
-                    s.Replace("BUILTIN/", "");
+                    s = s.Length == 8 ? "" : s.Substring(8);
```

Comment:
Umm, this is the same as `s = s.Substring(8);`, since s.Substring(s.Length) returns "".
And it cannot be less than 8 chars